### PR TITLE
#103 - purging doc blocks option for no comment fixer

### PR DESCRIPTION
**Pull request by kamilpiech97**
Added option for purging doc blocks in `NoComment` fixer. That option is set to `false` as default but purge mode has `true` value.

Before:

```
<?php

declare(strict_types=1);

use Illuminate\Support\Str;

/**
 * This is configuration file for Laravel application.
 * @see https://laravel.com/docs/11.x/database
 */
return [
    "default" => env("DB_CONNECTION", "mysql"),
]
```

After:

```
<?php

declare(strict_types=1);

use Illuminate\Support\Str;

return [
    "default" => env("DB_CONNECTION", "mysql"),
]
```

### CODE REVIEW

File: tests/codestyle/PurgeWithoutDocCommentsTest.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?php
```
Comment (krzysztofrewak) [inline tests/codestyle/PurgeWithoutDocCommentsTest.php:1]: I think I would add this to [PurgeTest.php](https://github.com/blumilksoftware/codestyle/blob/main/tests/codestyle/PurgeTest.php)

**Review by krzysztofrewak — APPROVED** [review]

File: tests/codestyle/PurgeWithoutDocCommentsTest.php (hunk 1 above)
Comment (kamilpiech97) [reply tests/codestyle/PurgeWithoutDocCommentsTest.php:1]: Why?
This is another test case where purge mode is tested but without removing the doc comments.

Comment (krzysztofrewak) [reply tests/codestyle/PurgeWithoutDocCommentsTest.php:1]: But it's still the same fixer. You can run two different methods in one test class.

Comment (kamilpiech97) [reply tests/codestyle/PurgeWithoutDocCommentsTest.php:1]: aahh, okay

**Review by krzysztofrewak — APPROVED** [review]

**Review by jsawo — APPROVED** [review]

**Pull request merged**